packages/analytics/metric-cards/src/enums/metric-card-size.enum.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Why is this enum here when you already have `MetricCardSize` in `constants.ts`?

Reply:
Unused file, i'll remove.

Context: I attempted to move the MetricCardSize to an `enum` earlier in the PR, and that caused the typecheck to explode. So moved it back to `constants.ts`.